### DISCUSSION TITLE
CSS Klasse für den Schatten um die Widgets

### DISCUSSION THREAD

**Discussion by mrkskhnr** [Zefau/ioBroker.jarvis#1632] [Q&A] [answered]
Wie kann ich an einem transparenten Widget auch den umlaufenden Schatten, der einen Rahmen andeutet, entfernen. In welcher Klasse muss ich da was eintragen?
Möchte die Widgets ganz plain auf dem Hintergrund haben. Danke im Voraus für jede Hilfe.

![Bildschirmfoto vom 2022-02-22 23-55-53](https://user-images.githubusercontent.com/80929315/155233596-c1b622fb-12de-4208-bbb1-773cc6fd5d9b.png)

**Comment by mcuiobroker** [comment, marked as answer]
Versuch mal: 
```
.paper {
  border-radius: 0px !important;
  box-shadow: 0 0 0 black;
}
```
@Zefau  Einschränkungsmöglichkeit auf .jarvis-widget-widgetid einbauen?

**Reply by mrkskhnr** [reply]
Das hat geklappt. Danke!
